pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools", "wheel", "setuptools-rust"]
 
 [project]
 name = "libcst"
-description = "A concrete syntax tree with AST-like properties for Python 3.5, 3.6, 3.7, 3.8, 3.9, and 3.10 programs."
+description = "A concrete syntax tree with AST-like properties for Python 3.8, 3.9, 3.10, 3.11, and 3.12 programs."
```

Comment:
Oh, we still support parsing 3.5+ documents, its just the runtime that must be 3.8+

Reply:
ah it's not clear if this refers to runtime version or document version. I'll change it to just consistent with the README :)